docs/.vitepress/config.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
       appId: '7DCTSU0WBW',
     },
     features,
+    langs: ['en-US', 'zh-CN', 'es', 'fr-FR', 'jp'],
```

Comment:
Can we not using hard coded? see the script I wrote in [crowdin-generate.ts](https://github.com/element-plus/element-plus/blob/dev/build/crowdin-generate.ts#L23) for getting it flexible.

Reply:
Yes, but not necessary.